optuna/study/study.py

```diff
@@ -439,10 +439,9 @@ def objective(trial):
                     :ref:`out-of-memory-gc-collect`
 
             show_progress_bar:
-                Flag to show progress bars or not. To disable progress bar, set this :obj:`False`.
-                Currently, progress bar is experimental feature and disabled
-                when ``n_trials`` is :obj:`None`, ``timeout`` is not :obj:`None`, and
-                ``n_jobs`` :math:`\\ne 1`.
+                Flag to show progress bars or not. To show progress bar, set this :obj:`True`.
+                Note that it is disabled when ``n_trials`` is :obj:`None`,
+                ``timeout`` is not :obj:`None`, and ``n_jobs`` :math:`\\ne 1`.
```

Comment:
Isn't it `or`, meaning we cannot use the progress bar if any one of them is `True`? 
```suggestion
                ``timeout`` is not :obj:`None`, or ``n_jobs`` :math:`\\ne 1`.
```

Reply:
https://github.com/optuna/optuna/blob/master/optuna/study/_optimize.py#L52
According to the current implementation, `and`  is correct. Iff we allow unlimited trial, set a time-limit, and make multiple jobs, then `progress_bar` is disabled.

Reply:
Thank you for the confirmation!